proxy/src/auth/backend/jwt.rs

```diff
@@ -409,14 +409,22 @@ impl JwkCacheEntryLock {
 
         if let Some(exp) = payload.expiration {
             if now >= exp + CLOCK_SKEW_LEEWAY {
-                return Err(JwtError::InvalidClaims(JwtClaimsError::JwtTokenHasExpired));
+                return Err(JwtError::InvalidClaims(JwtClaimsError::JwtTokenHasExpired(
+                    exp.duration_since(SystemTime::UNIX_EPOCH)
```

Comment:
Maybe it would be more useful to display in UTC format (to_rfc3339()) ?

Reply:
I thought it's easy enough to convert the numeric timestamp when needed. We can override `Display` for the error to print something else, sure.

Reply:
Let's stick with the u64 for now in the error variant for now and consider a custom formatting of that error later if really needed.